### ISSUE TITLE
allow sdss_install to install only module files without the repo

### ISSUE THREAD

**Issue by havok2063** [sdss/sdss_install#35]
sdss_install should be able to install module files for products without necessarily installing the repo.  This can be useful when trying to get module dependencies working but you don't really need, or plan to access, the repo itself.  

For example, the `mangadrp` module file depends on `platedesign` which depends on `platelist`, `plate` and four other repos/software that I don't want to install but I need the module files for.  

The old version of `Module` would still load the current modulefile even if it couldn't find dependency modulefiles, but with `Modules 4+` the modulefile fails to load.   

The `-L` option in `sdss_install` still depends on having the repository checked out locally, since it looks for the `.module` template file in `etc/`.

**Comment by joelbrownstein** [comment]
this works in sdss4tools, but would sdss4install also fail because Modules 4+ works differently with respect to warnings vs errors?

**Comment by havok2063** [comment]
I tried it with `sdss4install` and got the same problem.  They both fail in the same way and do not create the module file.  For example, trying to install the module file for the `transfer` product that I don't have installed locally gives

```
sdss4install -L repo/sdss/transfer trunk
sdss4install - INFO - Creating Modules directory transfer
sdss4install - INFO - Done! (failed modules)
sdss4install - INFO - Nonexistent template u'/Users/Brian/Work/sdss/repo/sdss/transfer/trunk/etc/transfer.module'

```

Is there a correct syntax where this would work?  

The Modules 4+ issue is only when (un)loading modules and their dependencies.

**Comment by joelbrownstein** [comment]
I suppose this use case was not part of the original feature request

**Comment by havok2063** [comment]
Yeah, I think `sdss(4)install` was designed to easily reinstall module files, with `-L`, for products already installed.  This issue is outlining a new feature enhancement to be added.
